src/bfx_tools/vaxrank.ml

```diff
@@ -0,0 +1,150 @@
+open Biokepi_run_environment
+open Common
+
+
+(*
+  $ vaxrank --help
+  usage: vaxrank [-h] --vcf VCF [--genome GENOME] --bam BAM
```

Comment:
Can vaxrank take in multiple VCFs? /cc @iskandr I thought you mentioned it could (or would?)?